plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/AbstractRefreshWithCustomizedStyleTest.java

```diff
@@ -415,14 +416,15 @@ protected void doTestStyleCustomizationThroughColorSelectionFromAppearanceSectio
 
             // Step 2: Enable button and check result
             buttonFromAppearanceSectionToTest.click();
-            assertTrue(BAD_CURRENT_COLOR_BUTTON, bot.button(defaultButtonInColorPalette[i]).isActive());
```

Comment:
You can also remove BAD_CURRENT_COLOR_BUTTON and defaultButtonInColorPalette if you remove this line.